src/controllers/article.js

```diff
@@ -3,6 +3,7 @@ import ArticleModel from '../helpers/articles';
 import { articleAverageRating } from './articleRatingController';
 import TagHelpers from '../helpers/tagHelpers';
 import FavoriteModelHelper from '../helpers/favorite';
+import logTracker from '../../logger/logTracer';
```

Comment:
@frostyblok Good job. I would love to know why you've imported this as `logTracker` when the module name is `logTracer` or is this a typo?

Reply:
This was completely a typographical error. I am going to make the necessary changes ASAP. Thank you for spotting this out.

Concerning your other feedback, I am happy with the fact you took time to go through the PR. But I believe I have something exactly like that in my implementation. For instance, every time we start the server, the logger logs it as an 'info', and when there is an error in the catch block, the logger logs it as 'error'

This is the logger in the server.js
<img width="500" alt="screen shot 2018-12-19 at 9 57 41 am" src="https://user-images.githubusercontent.com/40468678/50209717-b4727380-0374-11e9-82f6-db4e3b13b8fb.png">

And this is the logger we have in the catch block
<img width="629" alt="screen shot 2018-12-19 at 9 58 07 am" src="https://user-images.githubusercontent.com/40468678/50209732-bdfbdb80-0374-11e9-804f-9d734ed1e284.png">